pkg/security/ebpf/c/include/maps.h

```diff
@@ -28,7 +28,7 @@ BPF_ARRAY_MAP(syscall_ctx_gen_id, u32, 1)
 BPF_ARRAY_MAP(syscall_ctx, char[MAX_SYSCALL_CTX_SIZE], MAX_SYSCALL_CTX_ENTRIES)
 
 BPF_HASH_MAP(activity_dumps_config, u64, struct activity_dump_config, 1) // max entries will be overridden at runtime
-BPF_HASH_MAP(activity_dump_config_defaults, u32, struct activity_dump_config, 1)
+BPF_HASH_MAP(activity_dump_config_defaults, u32, struct activity_dump_config, 4)
```

Comment:
If I understand it correctly, shouldn't it be 5 in case systemd is enabled?

Reply:
Good catch, I did a last minute test and committed it by mistake